app/views/admin/memberships/show.html.haml

```diff
@@ -4,12 +4,24 @@
     %legend.govuk-fieldset__legend.govuk-fieldset__legend--xl
       %h1.govuk-fieldset__heading
         Unlink user from a supplier
+
 .govuk-grid-row
   .govuk-grid-column-two-thirds
-    %h2.govuk-heading-m= @user.name
-    %p= @user.email
-    %p This user will no longer be able to complete tasks on behalf of the following supplier
-    %h2.govuk-heading-m= @membership.supplier.name
+    %p
+      The user
+      = @user.name
```

Comment:
If they don't need the user's email to be shown, could you add a note in the commit message to say so? Just so know it wasn't removed by accident.

Reply:
Sure, I'll do my best 👍